affinity/cpp-23/d1436r3.md

```diff
@@ -454,6 +467,10 @@ The value returned from `execution::query(e1, memory_locality_intersection_t(e2)
 
 There are a number of additional features which we are considering for inclusion in this paper but are not ready yet.
 
+## Iteration space subdivision property
+
+It is defined in this proposal for the `bulk_execution_affinity_t` properties that when the size in an invocation of `execution::bulk_execute` is greater than the *available concurrency* then it is implementation how that iteration space is subdivided into a consecutive sequence of work-items. The authors of this proposal intend to propose a follow up property for specifying how an iteration space should be subdivided into chunks in this case.
```

Comment:
@AerialMantis "... then it is implementation defined how that iteration space..."